examples/nlp/token_classification/data/get_libritts_data.py

```diff
@@ -40,6 +40,68 @@
 }
 
 
+def remove_punctuation(word: str):
```

Comment:
Please move these methods to https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/data/token_classification/token_classification_utils.py, and both scripts can import from there